CMakeLists.txt

```diff
@@ -115,11 +115,17 @@ add_library(dso ${dso_SOURCE_FILES} ${dso_opencv_SOURCE_FILES} ${dso_pangolin_SO
 #set_property( TARGET dso APPEND_STRING PROPERTY COMPILE_FLAGS -Wall )
 
 
+if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin") # OSX
+    set(BOOST_THREAD_LIBRARY boost_thread-mt)
+else()
+    set(BOOST_THREAD_LIBRARY boost_thread)
+endif()
+
```

Comment:
P.S.: Just for the record, I noticed that cmake provides this as `Boost_THREAD_LIBRARY`, so we could have just used that... I will correct this at some point.